src/components/forums/forum/TopicList.tsx

```diff
@@ -59,20 +89,21 @@ export function TopicList({ forumData }: TopicListProps) {
         )}
       </div>
     </div>
-  );
+    );
+  }
 }
 
 interface RowContentProps {
   topic: ForumPost;
   forumData: ForumData;
 }
 
-function RowContent(props: RowContentProps) {
+function RowContent(props: RowContentProps): JSX.Element {
```

Comment:
see above about two components in a file

Reply:
in this case, it does make sense to separate them